lib/camerapreview.dart

```diff
@@ -195,23 +192,11 @@ class CameraAwesomeState extends State<CameraAwesome> with WidgetsBindingObserve
     selectedPreviewSize = null;
     selectedAndroidPhotoSize = null;
     _brightnessCorrectionDataSub?.cancel();
-    _permissionStreamSub?.cancel();
     _orientationStreamSub?.cancel();
     super.dispose();
   }
 
   Future<void> initPlatformState() async {
-    // wait user accept permissions to init widget completely on android
-    if (Platform.isAndroid) {
-      _permissionStreamSub = CamerawesomePlugin.listenPermissionResult()!.listen((res) {
-        if (res) {
-          initPlatformState();
-        }
-        if (widget.onPermissionsResult != null) {
-          widget.onPermissionsResult!(res);
-        }
-      });
-    }
     hasPermissions = await CamerawesomePlugin.checkPermissions();
     if (widget.onPermissionsResult != null) {
       widget.onPermissionsResult!(hasPermissions);
```

Comment:
And remove this code?

Reply:
Kept it in so that the caller knows if the permissions that are required for this lib to work are actually granted or not (the lib basically does nothing when permissions aren't given so there has to be a way for developers to know that). Currently unused by us but there still is a valid use case for this.